geospaas/nansat_ingestor/management/commands/ingest.py

```diff
@@ -55,4 +55,4 @@ def handle(self, *args, **options):
             elif type(ds)==Dataset:
                 self.stdout.write('%s has been added before.\n' % non_ingested_uri)
             else:
-                self.stdout.write('Could not add %s.\n' % non_ingested_uri)
+                self.stdout.write('Could not add %s. It may updated as an improvement of prior hervested version.\n' % non_ingested_uri)
```

Comment:
Not really needed here.
```suggestion
                self.stdout.write('Could not add %s.\n' % non_ingested_uri)
```